pkg/domain/domain.go

```diff
@@ -3329,6 +3343,38 @@ func (do *Domain) planCacheEvictTrigger() {
 	}
 }
 
+// SetupWorkloadBasedLearningWorker sets up all of the workload based learning workers.
+func (do *Domain) SetupWorkloadBasedLearningWorker() {
+	wbLearningHandle := workloadbasedlearning.NewWorkloadBasedLearningHandle()
+	// Start the workload based learning worker to analyze the read workload by statement_summary.
+	do.wg.Run(
+		func() {
+			do.readTableCostWorker(wbLearningHandle)
```

Comment:
If there are multiple TiDB instances in this cluster, can we guarantee there is only one node to execute this task?

Reply:
Yes no problem